bundles/org.eclipse.e4.ui.workbench.renderers.swt/src/org/eclipse/e4/ui/workbench/renderers/swt/CTabRendering.java

```diff
@@ -626,11 +626,11 @@ void drawSelectedTab(int itemIndex, GC gc, Rectangle bounds) {
 			if (onBottom) {
 				highlightOnTop = !highlightOnTop;
 			}
-			int highlightHeight = 2;
-			int verticalOffset = highlightOnTop ? 0 : bounds.height - (highlightHeight - 1);
+			int highlightHeight = superimposeKeylineOutline ? 3 : 2;
+			int verticalOffset = highlightOnTop ? 0 : outlineBoundsForOutline.height - (highlightHeight - 1);
```

Comment:
With `superImposeKeyline == true` this will now draw a line at the bottom that is one pixel higher than expected, won't it?